resources/META-INF/plugin.xml

```diff
@@ -10,7 +10,7 @@
   <vendor url="https://github.com/flutter/flutter-intellij">flutter.io</vendor>
 
   <category>Custom Languages</category>
-  
+  <version>29.0.1</version>
```

Comment:
I didn't think the version # was checked into `plugin.xml`?  (But I'm likely mis-remembering...)

Reply:
It looks like the plugin tool added it when it shouldn't have. I manually deleted it for now.